apps/web/src/server/views/applications/case-documentation/properties/documentation-properties.njk

```diff
@@ -56,20 +56,23 @@
                 </div>
                 <nav class="govuk-grid-row govuk-!-margin-top-0">
 					<h4 class="govuk-heading-m govuk-!-margin-left-3 govuk-!-margin-bottom-4">Document actions</h4>
-					<div class="govuk-grid-column-two-thirds govuk-!-margin-top-0">
+					<form action='../unpublishing-queue' method='post' class="govuk-grid-column-full govuk-!-margin-top-0">
 						{% set isNotDisabled = documentationFile.publishedStatus !== 'awaiting_upload' and documentationFile.publishedStatus !== 'awaiting_virus_check' and documentationFile.publishedStatus !== 'failed_virus_check' %}
 						{% set isPreviewActive = isNotDisabled and(documentationFile.mime === 'application/pdf' or documentationFile.mime === 'image/jpeg' or documentationFile.mime === 'image/png') %}
 
-            {% if isNotDisabled %}
-              <a class="govuk-button govuk-button--secondary govuk-!-margin-0" href="{{ 'document-download'|url({caseId: caseId, documentGuid: documentationFile.documentGuid, version: documentationFile.version, isPreviewActive: isPreviewActive}) }}">Open</a>
-              <a class="govuk-button govuk-button--secondary govuk-!-margin-0 govuk-!-margin-left-2" href="{{ 'document-download'|url({caseId: caseId, documentGuid: documentationFile.documentGuid, version: documentationFile.version}) }}">Download</a>
-            {% endif %}
+						{% if isNotDisabled %}
+							<a class="govuk-button govuk-button--secondary govuk-!-margin-0" href="{{ 'document-download'|url({caseId: caseId, documentGuid: documentationFile.documentGuid, version: documentationFile.version, isPreviewActive: isPreviewActive}) }}">Open</a>
```

Comment:
using a POST request instead of a just a link, allows us to re-use the same controller we use in the many-files scenario